Documentation/quickstart/debugging.rst

```diff
@@ -89,28 +89,31 @@ You can put a bunch of these into a simple script to configure the logging the w
    make oldconfig
 
 
-JTAG Debugging
---------------
+JTAG/SWD Debugging
+------------------
 
 `JTAG <https://en.wikipedia.org/wiki/JTAG>`_ is a set of standards that specify a way to attach a hardware device to
 your embedded board, and then remotely control the CPU. You can load code, start, stop, step through the program, and
-examine variables and memory.
-
-This guide assumes your board has a JTAG connector, and you have a JTAG hardware debugger like a
-`Segger J-Link <https://www.segger.com/products/debug-probes/j-link/>`_ or `OpenOCD <http://openocd.org/doc-release/html/index.html>`_.
+examine variables and memory. `SWD <https://en.wikipedia.org/wiki/JTAG#Similar_interface_standards>`_ is an
+Arm-specific, alternative 2-pin electrical interface that uses the same protocol and can be used too.
```

Comment:
Does it matter that the author of that wiki page claims a Creative_Commons_Attribution-ShareAlike license?

https://en.wikipedia.org/wiki/Wikipedia:Text_of__3.0_Unported_License

Wikipedia is also a registered trademark.  Does that matte?

Reply:
I can rewrite "alternative 2-pin electrical interface that uses the same protocol" which was copy-pasted from there. I didn't think of if given it was just a short sentence. Would that be better?

> Wikipedia is also a registered trademark. Does that matte?

In terms of the trademark, we only include the word Wikipedia in the actual link to its website. I don't know about legal, stuff but I would expect that to be OK? Otherwise it would be difficult to link anything. Does anyone one for certain?

Sony must be a trademark too. Should that be mentioned somewhere?

Thanks.